### DISCUSSION TITLE
Unable to change option color

### DISCUSSION THREAD

**Discussion by SagarPanchasara** [srivastavaanurag79/react-native-paper-select#36] [General]
is there any way to change the option text color
![image](https://github.com/srivastavaanurag79/react-native-paper-select/assets/11556571/0999b0bf-c23c-41a0-9fef-017a221abc78)

in this scenario where the background color is white, it is "White Text On A White Background"

**Comment by srivastavaanurag79** [comment]
Use `checkboxProps`  to change label style if you are on the latest version you can check for the type in the docs and `checkboxLabelStyle` if you are using version before 0.4.3
